hydrolib/core/dflowfm/mdu/models.py

```diff
@@ -815,6 +896,13 @@ class Comments(INIBasedModel.Comments):
             "Space separated list of files, containing information about observation cross sections.",
             alias="crsFile",
         )
+        foufile: Optional[str] = Field(
+            "Fourier analysis input file *.fou", alias="fouFile"
```

Comment:
Isn't it an output file?

Reply:
No, it's the Fourier configuration input file. The output file is `*_fou.nc`.